services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -962,6 +962,7 @@ async def remove_project_dynamic_services(
     app: web.Application,
     notify_users: bool = True,
     user_name: Optional[UserNameDict] = None,
+    suppress_project_locked_error: bool = False,
```

Comment:
I don't really like that new argument, I find the function gets more and more complex.
Why not just enhance the docstring, say that it can raise ProjectLockError instead? let the clients decide what they want to do by catching the exceptions or not.